x-pack/plugins/security_solution/public/common/components/ml/anomaly/use_anomalies_table_data.ts

```diff
@@ -121,13 +121,17 @@ export const useAnomaliesTableData = ({
       isSubscribed = false;
       abortCtrl.abort();
     };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [
+    // eslint-disable-next-line react-hooks/exhaustive-deps
     influencersOrCriteriaToString(influencers),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
     influencersOrCriteriaToString(criteriaFields),
     startDate,
     endDate,
     skip,
     userPermissions,
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
It might be worth improving this, as sorting something each render could hinder performance.